runbook.go

```diff
@@ -131,7 +131,7 @@ func flattenYamlAliases(in []byte) ([]byte, error) {
 		goyaml.Flow(false),
 		goyaml.UseSingleQuote(false),
 		goyaml.UseLiteralStyleIfMultiline(false),
-		goyaml.IndentSequence(true),
+		goyaml.IndentSequence(false),
```

Comment:
Am I correct in understanding that indentation is not necessary because it is in the middle of the process?

Reply:
Yes. We want to eliminate indentation during encoding and reduce the final byte size!